src/apps/authentication/__test__/components/UserTokenDetails.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import {render} from "../../../../test-utils";
+import {UserTokenDetails} from "../../components";
+import {testToken} from "../test_data";
+
+type userTokenDetailsProps = React.ComponentProps<typeof UserTokenDetails>;
+
+const baseProps: userTokenDetailsProps = {
+    token: testToken
+};
+
+function renderUI(props: Partial<userTokenDetailsProps> = {}) {
+    return render(<UserTokenDetails {...baseProps} {...props} />);
+}
+
+describe("UserTokenDetails", () => {
+    it("should match snapshot", () => {
+        const {container} = renderUI(baseProps);
+
+        expect(container).toMatchSnapshot();
+    });
+});
```

Comment:
Apart from doing snapshot test, you can add one test to validate correct token

Reply:
Hi Rakesh, we didn't add new tests as this is a view page and the details we be covered as part of snapshot.Hope that is fine